src/components/common/StoreReview.tsx

```diff
@@ -0,0 +1,141 @@
+import { useEffect, useState } from 'react';
+import * as ExpoStoreReview from 'expo-store-review';
+import { Trans, t } from '@lingui/macro';
+import { differenceInDays } from 'date-fns';
+
+import { showToast } from './Toaster';
+
+import ImprovementForm from '~components/store-review/ImprovementForm';
+import {
+  BottomSheet,
+  FillButton,
+  OutlineButton,
+  Stack,
+  Text,
+} from '~components/uikit';
+import storage from '~utils/storage';
+import { styled } from '~styles';
+
+const DAYS_TO_WAIT_BEFORE_ASKING_AGAIN = 7; // 7 days
+const DAYS_BETWEEN_REVIEWS = 365; // 1 year
+
+export default function StoreReview() {
+  const [open, setOpen] = useState<boolean>(false);
```

Comment:
NIT: remove unnecessary type generics `<boolean>` as they can be inferred from the initial value.